geth/api/app_state.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"fmt"
+	"strings"
+)
+
+// AppState represents if the app is in foreground, background or some other state
+type AppState string
+
+func (a AppState) String() string {
+	return string(a)
+}
+
+// Specific app states
+// see https://facebook.github.io/react-native/docs/appstate.html
+const (
+	AppStateForeground = AppState("foreground")
+	AppStateBackground = AppState("background")
+	AppStateInactive   = AppState("inactive")
+
+	AppStateInvalid = AppState("")
+)
+
+// validAppStates returns an immutable set of valid states.
+func validAppStates() []AppState {
```

Comment:
Maybe declare it as a `var`?

Reply:
there is [a discussion about it]y(https://github.com/status-im/status-go/pull/735#discussion_r174515431) in this review already.
in general, I don't like that `var` is mutable.